src/arrayMethodReduce.js

```diff
@@ -1,12 +1,27 @@
-'use strict';
+"use strict";
 
 /**
  * Implement method Reduce
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let total = initialValue;
```

Comment:
```suggestion
    let prevValue = initialValue;
```
total is quite a misleading name in this case.

Reply:
you can call it just 'index'